### ISSUE TITLE
Messages posted to Discord are not accessible by screen readers

### ISSUE THREAD

**Issue by parhamdoustdar** [Playabl-io/playabl#287]
Right now, the messages that Playabl sends to Discord are not readable on iOS and MacOS. Not sure about other platforms since I'm a Mac user.

To reproduce:

1. Turn Voiceover on on MacOS or iOS.
2. Open Discord, and go to the channel where Playabl notifications are sent.
3. Press F6 until you land on the list of messages, and navigate through the messages using the up/down arrow keys. Note that the content of the messages isn't spoken.

- OS: Sequoia 15.5
- Discord version stable 418516 (f170514) Host 0.0.352 arm64 Build Override: N/A macOS 15 (24.5.0)

**Smartphone (please complete the following information):**

- Device: iPhone 15 Pro
- OS: iOS 18.5

**Additional context**
I suspect the cause of this is actually Discord. More specifically, embeds seem to be unreadable. The solution might be to find another alternative to embeds that helps us post images, links, and anything else that is a part of a notification, but in a way that is communicated well by Discord to screen readers.

I have already reported this issue to Discord in the meantime.
